NLeSC/python-template
# add Zenodo integration issue template

Fixes #235 

**Important Note**:
Please use https://sandbox.zenodo.org/ instead of https://www.zenodo.org/ to test the instructions otherwise entry you create on Zenodo cannot be removed easily.

**Summary**
This pull request adds an issue template for Zenodo integration.

**Testing**
- Create a new package
- Push the package to GitHub
- Check the generated issue about Zenodo integration
- Follow the instructions on https://sandbox.zenodo.org/
